app/representers/api/msg/series_representer.rb

```diff
@@ -0,0 +1,11 @@
+# encoding: utf-8
+
+class Api::Msg::SeriesRepresenter < Api::SeriesRepresenter
+  # don't embed the stories, makes this a much smaller message
+  embed :public_stories,
+        as: :stories,
+        paged: true,
+        item_class: Story,
+        item_decorator: Api::Min::StoryRepresenter,
+        zoom: false
```

Comment:
so it's the `zoom: false` that dictates stories won't be embedded by default, right? and then if the representer is passed the zoom param, then it'll embed stories?